docs/docs/how-tos/index.md

```diff
@@ -6,6 +6,7 @@ Welcome to the LangGraph how-to guides! These guides provide practical, step-by-
 
 The core guides show how to address common needs when building out AI workflows, with special focus placed on [ReAct](https://arxiv.org/abs/2210.03629)-style agents with [tool calling](https://python.langchain.com/docs/modules/model_io/chat/function_calling/).
 
+- [ReAct agent](create-react-agent.ipynb): How to create a simple ReAct agent
```

Comment:
For the description, it may be more helpful to word it as something like? 

"How to create a simple tool-calling agent"
Less redundant . Maybe we even say what react means (How to create a tool-calling agent that **Re**asons and **Act**s to accomplish tasks)